test-e2e/panels/filters.test.ts

```diff
@@ -79,3 +84,32 @@ test('process 1d FID 13c spectrum', async ({ page }) => {
     await baselineCorrectionFilter(nmrium);
   });
 });
+
+test('Processing 13c spectrum', async ({ page }) => {
```

Comment:
You don't need to create a new test but just append the new steps to the previous step.

Reply:
but in the previous test there is more filter used , can i remove them?

Reply:
You may leave them if there are more filters.

Reply:
okey thank you